flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java

```diff
@@ -217,24 +258,27 @@ public boolean doConvert(
                     return false;
                 }
                 TinyIntVector tinyIntVector = (TinyIntVector) fieldVector;
-                fieldValue = tinyIntVector.isNull(rowIndex) ? null : tinyIntVector.get(rowIndex);
-                addValueToRow(rowIndex, fieldValue);
+                Object fieldValueTiny =
```

Comment:
Does this need to be changed? We can just reuse the fieldValue variable at the outermost level